packages/php/build.sh

```diff
@@ -4,16 +4,16 @@ TERMUX_PKG_LICENSE="PHP-3.01"
 TERMUX_PKG_LICENSE_FILE=LICENSE
 TERMUX_PKG_MAINTAINER="@termux"
 # Please revbump php-* extensions along with "minor" bump (e.g. 8.1.x to 8.2.0)
-TERMUX_PKG_VERSION="8.3.10"
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_VERSION="8.4.1"
+TERMUX_PKG_REVISION=1
```

Comment:
Package revisions on Termux start at 0.
Please remove `TERMUX_PKG_REVISION`
```suggestion
```

Reply:
libpq-fe.h not found